source/templates.tex

```diff
@@ -138,7 +138,7 @@
 \begin{note}
 Default arguments for function templates and for member functions of
 class templates are considered definitions for the purpose of template
-instantiation\iref{temp.decls} and must obey the one-definition rule\iref{basic.def.odr}.
+instantiation\iref{temp.decls} and thus the one-definition rule\iref{basic.def.odr} applies.
```

Comment:
Please add a comma before "and".

Reply:
@jensmaurer Would you have a moment for this ^^^?

Reply:
Fixed.